views/info/filters.pug

```diff
@@ -8,76 +8,96 @@ block content
   dl.row
     .accordion#syntax-accordion
       .card
-        #general-syntax.card-header(data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne")
-          button(class="btn btn-link" type="button" )
+        #general-syntax.card-header(data-toggle="collapse" data-target="#collapseGeneral" aria-expanded="true" aria-controls="collapseGeneral")
+          button(class="btn btn-link" type="button")
             h5 General
-        #collapseOne.collapse.show(aria-labelledby="general-syntax" data-parent="#syntax-accordion")
-          p You can combine any number of filters together, using AND, or OR.
-          p #[code t:instant or t:sorcery] cards that are either #[code instants] or #[code sorceries].
-          p #[code t:instant t:tribal] cards that are both #[code instants] and #[code tribal].
-          p You can also use quotes to ignore spaces.
-          p #[code goblin blood] cards whose names contain both #[code blood] and #[code goblin].
-          p #[code "goblin blood"] cards whose names contain #[code goblin blood].
-          p #[code o:destroy o:target o:creature] cards whose oracle text contains #[code destroy] and #[code target] and #[code creature].
-          p #[code o:"destroy target creature"] cards whose oracle text contains #[code destroy target creature].
-          p You can also use parentheses.
-          p #[code t:instant OR (t:creature o:flash)] cards which are instants, or cards which are creatures with flash.
-          p #[code (t:artifact t:creature) OR (t:instant OR t:sorcery)] cards which are instants or sorceries, or cards which are artifact creatures.
-          p You can put - before anything to negate it.
+        #collapseGeneral.collapse.show(aria-labelledby="general-syntax" data-parent="#syntax-accordion")
+          p You can combine any number of filters together using #[code AND] or #[code OR].
+          p #[strong Examples:]
+          table.table
+            tr
+              td(scope="col") #[code t:instant OR t:sorcery]
+              td(scope="col") Cards that are either instants or sorceries.
+            tr
+              td(scope="col") #[code t:instant t:tribal]
+              td(scope="col") Cards that are both instants and tribal.
+          p You can use quotes to ignore spaces.
+          p #[strong Examples:]
+          table.table
+            tr
+              td(scope="col") #[code goblin blood]
+              td(scope="col") Cards whose names contain both "blood" and "goblin".
+            tr
+              td(scope="col") #[code "goblin blood"]
```

Comment:
This doesn't work currently you have to do name:"goblin blood"

Reply:
```suggestion
              td(scope="col") #[code name:"goblin blood"]
```